Added jobs referral work flow graph.

quintuschen opened the pull request:
I will put the graph into README.md once it is merged.

alex-zige commented:
Looks good!

Just a thought, would that be better to split to 2 flows for seeker and job poster? To simplify things a bit;
- if you are a job poster: [Flow 1]
- if you are a job seeker: [Flow 2]

CITANZ internal follow flow can be opaque from the eyes of the two parties above;

quintuschen commented:
When I included CITA in the graph, I thought job posters and job seekers might be interested in how CITA is involved in this. Since job posters need to notify us of the jobs' status, and job seekers will expect the follow-up procedure from us too, CITA will be visible in both of the flows anyway. However, I will create another two flows for job posters and job seekers.

openstacker commented:
Overall it looks very good. Thanks.

I think it's OK to use one diagram to demonstrate the overall workflow. Here are some comments from my view:

1. It's not necessary to mention detailed work, e.g. "Prepares CV", "Prepares JD", hence those works are well-known and routine.
2. The survey needs to be done by the referrer since it's hard to know the information of the job seekers from CITA side.

Thanks for the great work again. Cheers.

quintuschen commented:
Updated. Now we have two flows for each group while we keep the overall graph. Adjusted a few wordings and updated the overall workflow. I personally prefer the overall one since it covers how CITA is involved and does show clear interactions in terms of timeline.

alex-zige commented:
Good work, lets get this baby merged in first.
